docs/manual/system/settings.de.md

```diff
@@ -600,7 +600,7 @@ dazu auch die Informationen in der offiziellen [Symfony Dokumentation][SymfonyMa
 Falls der Benutzername oder das Passwort Sonderzeichen verwendet, müssen diese "URL enkodiert" werden. Es gibt
 verschiedene Online-Services, mit denen man auf einfache Weise eine beliebige Zeichenfolgen URL-encoden kann, z. B.
 [urlencoder.org](https://www.urlencoder.org/). Enkodiere den Benutzernamen und das Passwort separat, nicht gemeinsam
-mit dem Doppelpunkt.
+mit dem Doppelpunkt. Die jeweiliege Kodierung eines Sonderzeichen muß mit % eingeleitet werden: Also z.B. %%23 für das Sonderzeichen #.
```

Comment:
That depends on where you use it. A `%` character only needs to be doubled in your YAML config - but not if you use environment variables (via your `.env.local` for example). See https://symfony.com/doc/4.4/configuration.html#configuration-parameters

Reply:
However, should this not already be explicitly distinguished so?

Reply:
Add a hint for config.yml usage

Reply:
I don't know what you mean? You need to distinguish that in your sentence that you want to add ;). e.g.:

> Note that in your YAML configuration files any `%` character [needs to be escaped](https://symfony.com/doc/4.4/configuration.html#configuration-parameters) with another `%`, so `%%23` for example for the the special character `#`. This will not be necessary if you defined your DSN in an environment variable and you are only referencing that variable in your YAML configuration.